.pre-commit-config.yaml

```diff
@@ -29,6 +29,11 @@ repos:
       - id: requirements-txt-fixer
       - id: trailing-whitespace
 
+  - repo: https://github.com/asottile/setup-cfg-fmt
```

Comment:
If the order is not otherwise important then, please keep the hooks sorted alphabetically so that it is easier to spot missing hooks and more difficult to insert duplicates.  `setup-cfg-fmt` would come last in this file.